frontend/resourceadm/components/ResourceDeployEnvCard/ResourceDeployEnvCard.module.css

```diff
@@ -1,10 +1,11 @@
 .cardWrapper {
-  display: flex;
+  display: inline-flex;
   flex-direction: column;
   align-items: flex-start;
-  width: 16rem;
+  width: 38%;
```

Comment:
Is this number supposed to be hardcoded?

Reply:
changed to grid layout to be more dynamic